### DISCUSSION TITLE
New translation in Nederlands (Dutch)

### DISCUSSION THREAD

**Discussion by mtdvlpr** [sircharlo/meeting-media-manager#166] [Translations]
I would like to help to translate JWMMF into a language I speak, Nederlands (Dutch) (O/nl).

**Comment by sircharlo** [comment]
Thanks for reaching out @mtdvlpr !
If you're comfortable forking a GitHub repo, feel free to do so and submit a PR with a `nl.json` file added to the `locales` folder, with the contents copied from `en.json` and translated into your language.
If not, feel free to simply download [the english locale file](https://github.com/sircharlo/jw-meeting-media-fetcher/blob/master/locales/en.json) from [here](https://raw.githubusercontent.com/sircharlo/jw-meeting-media-fetcher/master/locales/en.json), translate the strings, and save and upload your translation in this discussion. 
Thanks for your willing spirit!

**Reply by sircharlo** [reply]
Implemented in #167

**Reply by sircharlo** [reply]
Released in #173
